.github/workflows/build-desktop-app-installers.yml

```
@@ -0,0 +1,181 @@
+      # These persist into the composite action's gradle invocation via GITHUB_ENV.
+      - name: "Enable Compose macOS signing"
+        if: matrix.platform == 'macos' && inputs.sign
+        env:
+          SIGNING_IDENTITY: ${{ secrets.MACOS_DEVELOPER_ID_SIGNING_IDENTITY }}
+        run: |
+          set -euo pipefail
+          echo "MACOS_SIGN=true" >> "$GITHUB_ENV"
+          printf 'MACOS_DEVELOPER_ID_SIGNING_IDENTITY=%s\n' "$SIGNING_IDENTITY" >> "$GITHUB_ENV"
+
+      - name: "Package installer"
+        uses: ./.github/actions/gradle-task-run
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/gradle-task-run": line 11: cannot unmarshal !!str `false` into bool

(action)